java/core/src/main/java/com/microsoft/bond/StructBondType.java

```diff
@@ -2053,6 +2197,10 @@ public final void serialize(
             this.verifyFieldWasNotYetDeserialized(wasAlreadyDeserialized);
             return this.fieldType.deserializeSomethingField(context, this);
         }
+
+        public final SomethingObject<String> deserialize(UntaggedDeserializationContext context) throws IOException {
+            return SomethingObject.wrap(this.fieldType.deserializeValue(context));
```

Comment:
I don't remember doing this intentionally. IDE auto-cleanup funkiness, maybe?

Changed all instances to `Something.wrap()`.